setup.cfg

```diff
@@ -0,0 +1,13 @@
+[check-manifest]
+ignore =
+    */test
+    */test/*
+    .*
+    node_modules
+
+[pep257]
+ignore = D202
+explain = true
+
+[yapf]
+based_on_style = pep8
```

Comment:
@seanh - We have a config for it in H. Nick and I had some discussion about this. I personally advocate having a tool automate formatting entirely, even if it sometimes produces output that isn't exactly what one might write by hand. I hope I'm characterizing Nick's view accurately by saying that he saw it as a useful tool but that the output was not sufficiently high quality to always use it without any adjustment.

Running it over a Python script is a good way to get it into reasonable shape and correct obvious errors IMO.